Dockerfile

```diff
@@ -0,0 +1,24 @@
+# stage-node generates front-end files
+FROM node:8-stretch AS stage-node
+RUN mkdir -p /app/src
+ADD . /app/src/
+WORKDIR /app/src
+RUN npm install
+RUN npm run build:production
+
+# main
+FROM python:3.6-stretch
+COPY --from=stage-node /app/src /app/src
+WORKDIR /app/src
+
+RUN apt-get update \
```

Comment:
Could you please use just one `RUN` like https://github.com/vijos/jd4/blob/master/Dockerfile#L3 ?